creator-node/src/resizeImage.js

```diff
@@ -0,0 +1,92 @@
+const Jimp = require('jimp')
+const ExifParser = require('exif-parser')
+
+const CRAZY_HEIGHT = 6000 // No image should be taller than this.
+const COLOR_WHITE = 0xFFFFFFFF
+const IMAGE_QUALITY = 90
+const MIME_TYPE_JPEG = 'image/jpeg'
+
+/**
+ * Returns an image that's been resized, cropped into a square, converted into JPEG, and compressed.
+ * @param {Object} req
+ * @param {string} imageBuffer the buffer of the image to use
+ * @param {number} maxWidth max width of the returned image (default is 1,000px)
+ * @param {boolean} square whether or not to square the image
+ * @return {Buffer} the converted image
+ */
+async function resizeImage (req, imageBuffer, maxWidth, square) {
+  // eslint-disable-next-line
+  let exif
+  try {
+    exif = ExifParser.create(imageBuffer).parse()
+  } catch (error) {
+    req.logger.error(error)
+    exif = null
+  }
+
+  let img = await Jimp.read(imageBuffer)
+  
+  img = _exifRotate(img, exif)
```

Comment:
i'm wondering now if this line could just be
`_exifRotate(img, exif)`
and that function doesn't even need to return img